components/Navigation.tsx

```diff
@@ -113,10 +113,12 @@ function NavigationSubLink({ href, children }: NavigationSubLinkProps) {
   );
 
   return (
-    <span className="flex items-center ml-4">
+    <span className="relative flex items-center ml-4">
       <Link href={href}>
         <a className={classes}>
-          {router.asPath === href ? <Caret className="mr-1 w-2 h-2" /> : null}
+          {router.asPath === href ? (
+            <Caret className="absolute -ml-4 w-2 h-2" />
```

Comment:
do you need the relative / absolute for this?

Reply:
I do to not have to do magic margins